Documentation/network/concepts/masquerading.rst

```diff
@@ -44,6 +44,21 @@ Setting the routable CIDR
 Setting the masquerading interface
   See :ref:`masq_modes` for configuring the masquerading interfaces.
 
+Masquerade traffic to Remote Nodes
+**********************************
+
+To masquerade traffic to remote nodes in BPF masquerading mode, use the option ``enable-remote-node-masquerade: "true"``. 
+This option requires ``enable-bpf-masquerade: "true"`` and also either 
+``enable-ipv4-masquerade: "true"`` or ``enable-ipv6-masquerade: "true"`` to SNAT traffic for IPv4 and IPv6, respectively.
+This option only affects traffic from an Endpoint directed towards the address of a remote node, and not traffic between Endpoints on different nodes.
+
+This flag currently masquerades traffic to node ``InternalIP`` addresses. This may change in future. See : (:gh-issue:`35823`) and (:gh-issue:`17177`) for further discussion on this topic.
```

Comment:
Minor nit for formatting:
```suggestion
This flag currently masquerades traffic to node ``InternalIP`` addresses. This may change in future. See :gh-issue:`35823` and :gh-issue:`17177` for further discussion on this topic.
```